spec/models/registration_spec.rb

```diff
@@ -70,7 +70,6 @@
     registration.user = user
     expect(registration).to be_valid
   end
-
```

Comment:
Why did this poor blank line have to die? :sob:

Reply:
I eat blank lines for breakfast (fixed)